### PR TITLE
Blog Post: Contributing to the Development Guide

### PR DESCRIPTION
**Pull request by pymander**
Erik L. Arneson (that's me!) writes all about his experience contributing to the Development Guide. This commit includes edits from
Joel Barker.

### CODE REVIEW

File: content/en/blog/2020/contributing-to-the-development-guide/index.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+"The Kubernetes Code of Conduct is in effect, so please be excellent to each other." &mdash; Jorge
+Castro, SIG ContribEx co-chair
+{{< /imgproc >}}
+
+My fears were unfounded. Immediately, I felt welcome. I like to think this isn't just because I was
+working on a much needed task, but rather because the Kubernetes community is filled
+with friendly, welcoming people. During the weekly SIG ContribEx meetings, our reports on progress
+with the Development Guide were included immediately. In addition, the leader of the meeting would
+always stress that the [Kubernetes Code of Conduct]({{< ref "/resources/code-of-conduct" >}}) was in
+effect, and that we should, like Bill and Ted, be excellent to each other.
```
Comment (parispittman) [inline content/en/blog/2020/contributing-to-the-development-guide/index.md:53]: lol!

**Comment by parispittman** [comment]
Y'all were my last best thing before leaving G! :D haha love it! thanks so much for your efforts, including this great story! hope to see you around the community. :) vacation first!

File: content/en/blog/2020/contributing-to-the-development-guide/index.md (hunk 1 above)
Comment (castrojo) [reply content/en/blog/2020/contributing-to-the-development-guide/index.md:53]: I agree, lol!

**Comment by mrbobbytables** [comment]
The post is wonderful! 
Not to take away content from here, but what do you think of actually posting it over on the main kubernetes.io blog page? :) It'd get a lot more eyes on it all.

**Comment by pymander** [comment]
> The post is wonderful!

Thank you!

> Not to take away content from here, but what do you think of actually posting it over on the main kubernetes.io blog page? :) It'd get a lot more eyes on it all.

What about cross-posting it? You can always have a notice at the top saying "This originally appeared on the Kubernetes Contributor Blog." Then include a link to the original, and suddenly the contributor website gets more traffic!

**Pull request merged**